### ISSUE TITLE
Package.getStylesheetsPath is deprecated.

### ISSUE THREAD

**Issue by maarek** [indiejames/atom-iex#18]
Store package style sheets in the `styles/` directory instead of `stylesheets/` in the `iex` package

```
Package.getStylesheetsPath (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:444:9)
Package.getStylesheetPaths (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:455:32)
Package.loadStylesheets (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:437:38)
<unknown> (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:167:19)
```

**Comment by indiejames** [comment]
Just published an update to address this. Thanks for the heads up!

**Comment by indiejames** [comment]
Fixed in commit 28d6949.
